### ISSUE TITLE
about image input settings

### ISSUE THREAD

**Issue by EcustBoy** [opendilab/InterFuser#49]
Hi ~author, I wanna ask is the foucs view image input really useful for perception(e.g. traffic light detection)? have you done some ablation study about that? I'm curious about how much is the impact to light state perception if the focus view input is removed.

 I'll thanks for your reply~

**Comment by deepcs233** [comment]
Yes, we have conduct related ablation studies, please refer to Table 4 in our CoRL paper. It's useful for detecting traffic light and doesn't bring in any more extra flops or model weights.
